sympy/solvers/tests/test_ode.py

```diff
@@ -440,8 +440,8 @@ def test_nonlinear_2eq_order1():
     eq3 = (Eq(diff(x(t),t), y(t)*x(t)), Eq(diff(y(t),t), x(t)**3))
     tt = S(2)/3
     sol3 = [
-        Eq(x(t), 6**tt/(6*(-sinh(sqrt(C1)*(C2 + t)/2)/sqrt(C1))**tt)),
-        Eq(y(t), sqrt(C1 + C1/sinh(sqrt(C1)*(C2 + t)/2)**2)/3)]
+        Eq(x(t), 6**tt/(6*(sinh(sqrt(C1)*(C2 + t)/2)/sqrt(C1))**tt)),
+        Eq(y(t), -sqrt(C1 + C1/sinh(sqrt(C1)*(C2 + t)/2)**2)/3)]
```

Comment:
Are these solutions equivalent? Why has this changed? Does checksysodesol work for this ODE now?

Reply:
`checkodesol` is not working. I think the solution is equivalent to the previous one.

Reply:
Why has it changed?